.pylintrc

```diff
@@ -0,0 +1,24 @@
+[GENERAL]
+# disable=R0205, F0010, C0330, E1136, E0401
+disable=R0205, F0010, E1136, E0401
```

Comment:
I think you can spell these out instead of using the codes. That would help a lot, noone remembers what each of these means..

Reply:
Good point. :+1: 

While doing that, I see that only R0205 (useless-object-inheritance) is still relevant. Since we only support python3, this can be easily fixed. So the entries can all be removed now.